src/containers/ViewPage/ViewPage.tsx

```diff
@@ -10,14 +10,14 @@ interface IProps {
 }
 
 const ViewPage = (props: IProps) => {
+  const { data } = props
   const movieID = useSelector((state: stateTypes) => state.movieID)
-  const chosenMovie: moviesType = props.data.find(
-    (item) => item.id === movieID
-  )!
-
-  const sameGenreMovie = (): moviesType[] =>
-    props.data.filter(item =>
-      item.genres.every(element => chosenMovie.genres.includes(element))
+  const selectedMovie: moviesType = data.find((item) => item.id === movieID)!

```

Comment:
```suggestion
  const selectedMovie: Movie = movies.find((movie) => movie.id === movieID)!

```
Компонент вроде не абстрактный и не общий, тогда зачем использовать в наименовании  дата, итем и тд, если контекст четкий

Reply:
Найс хак с восклицательным знаком)))